server/server.go

```diff
@@ -64,19 +75,32 @@ func NewRpcEndPointServer(logger log.Logger, version, listenAddress, relayUrl, p
 	FlashbotsRPC = flashbotsrpc.New(relayUrl)
 	// FlashbotsRPC.Debug = true
 
-	return &RpcEndPointServer{
+	mux := http.NewServeMux()
```

Comment:
**explanation:**

move the routing setup into the `NewRpcEndPointServer` method.  this will make it possible to run more than one server (on different ports, for example).  also, it's nice to encapsulate the creation logic into one place.